Mjml.Net/Helpers/ConditionalTags.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Mjml.Net.Helpers;
```

Comment:
Just a minor thing: We should use a consistent syntax for namespaces.

Reply:
Agreed. However, see my comment below regarding changes to this file.